sys/android-architecture.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+	echo "Please enter an android architecture..."
+fi
+
+
+case "$1" in
+	aarch64)
+		cd `dirname $PWD/$0`
+		export CFLAGS="-O3"
+		./android-shell.sh aarch64 ./android-build.sh aarch64-static
+		;;
+	arm)
+		cd `dirname $PWD/$0`
+		export CFLAGS="-O3"
+		./android-shell.sh arm ./android-build.sh arm-static
+		;;
+	mips)
+		cd `dirname $PWD/$0`
+		export CFLAGS="-O3 -fPIC"
+		./android-shell.sh mips ./android-build.sh mips-static
+		;;
+	mips64)
+		cd `dirname $PWD/$0`
```

Comment:
lot of spaguetti here. you can just check arm|mips|mips64... in a single case and then use $arch to replace the part that is different for each target